### PR TITLE
Updated collection schema definitions to match public equivalents

### PR DESCRIPTION
**Pull request by kunagpal**

### CODE REVIEW

File: lib/schema/auth.json [hunk 1]
```diff
@@ -9,62 +9,86 @@
...
                 "oauth1",
-                "oauth2"
+                "oauth2",
+                "ntlm"
             ]
         },
         "awsv4": {
             "type": "array",
+            "title": "AWS Signature v4",
+            "description": "This helper attributes for [AWS Auth](http://docs.aws.amazon.com/AmazonS3/latest/dev/RESTAuthentication.html)",
```
Comment (kamalaknn) [inline lib/schema/auth.json]: Typo

File: lib/schema/auth.json [hunk 2]
```diff
@@ -104,113 +166,178 @@
...
+                            },
+                            "type": {
+                                "type": "string",
+                                "enum": ["string"]
                             }
                         }
                     },
                     {
                         "type": "object",
+                        "description": "The Digest server challenges clients. A nonce is a part of that challenge, stored in this field.",
```
Comment (kamalaknn) [inline lib/schema/auth.json]: Digest server?

File: lib/schema/auth.json [hunk 3]
```diff
@@ -463,101 +820,265 @@
...
                             }
                         }
                     },
                     {
                         "type": "object",
+                        "description": "This field contains the URL that has to be hit when the authentication process has completed.",
                         "properties": {
                             "key": {
                                 "type": "string",
                                 "enum": ["callBackUrl"]
```
Comment (kamalaknn) [inline lib/schema/auth.json]: callbackUrl ?

**Comment by kamalaknn** [comment]
@kunagpal LGTM. I've left a few comments, but those have to be made in both the repositories I beleive.

File: lib/schema/auth.json (hunk 3 above)
Comment (kunagpal) [reply lib/schema/auth.json]: Yes, this is one of many quirks: https://github.com/postmanlabs/postman-collection-transformer/blob/develop/lib/util.js#L289

**Pull request merged**